runtime/compiler/CMakeLists.txt

```diff
@@ -263,6 +263,10 @@ if(OMR_OS_LINUX OR OMR_OS_OSX)
 		-fno-threadsafe-statics
 		-Wno-invalid-offsetof
 	)
+	if(OMR_ENV_DATA32)
+		# avoid memset warnings, for warnings as errors
+		list(APPEND J9_CXXFLAGS -Wno-stringop-overflow>)
```

Comment:
I don't think you intended to include the `>` character (though it's unclear why builds don't fail).